swagger.json

```diff
@@ -10,7 +10,7 @@
     "title": "Anträge API",
     "version": "2.89"
```

Comment:
@gruenewa Please update the version to the internal repository, regenerate the Swagger files and index HTML, and create a new PR here.